src/codeu/chat/client/commandline/Chat.java

```diff
@@ -337,6 +349,238 @@ private ConversationContext find(String title) {
       }
     });
 
+    // U-INT-ADD (add user interest)
+    //
+    // Add a command that will add a user to the current user's
+    // interests when the user enters "u-int-add" while on the user panel
+    //
+    panel.register("u-int-add", new Panel.Command() {
+      @Override
+      public void invoke(List<String> args) {
+        for(String token : args){
```

Comment:
the for loop allows the user to add more than one interest at a time, I thought that would be quicker and more convenient, but should I limit it to one interest at a time?